### PR TITLE
chore(Breadcrumb): updated a11y docs

### LINKED ISSUE #3280: Update a11y docs for Breadcrumb

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #3280

### CODE REVIEW

**Review by nicolethoen — APPROVED** [review]
LGTM!

File: packages/v4/patternfly-docs/content/accessibility/breadcrumb/breadcrumb.md [hunk 1]
```diff
@@ -3,13 +3,70 @@ id: Breadcrumb
...
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-2" label="Standard keyboard navigation can be used to navigate between each breadcrumb item or other focusable elements." description={<span><kbd>Tab</kbd> navigates to the next breadcrumb item or focusable element, and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous breadcrumb item or focusable element.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-3" label="Users are able to tell whether a breadcrumb item will open in a new tab or window." description={<span>This can be best achieved by rendering an icon for visual context, and passing in visually-hidden text via the <code className="ws-code">pf-screen-reader</code> class.</span>} />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-4" label="Users are able to tell whether a breadcrumb item will open in a new tab or window." description={<span>This can be best achieved by rendering an icon for visual context, and passing in visually-hidden text via the <code className="ws-code">pf-screen-reader</code> class.</span>} />
+  </ListItem>
```
Comment (srambach) [inline packages/v4/patternfly-docs/content/accessibility/breadcrumb/breadcrumb.md:43]: I think this is a duplicate item?

File: packages/v4/patternfly-docs/content/accessibility/breadcrumb/breadcrumb.md [hunk 2]
```diff
@@ -3,13 +3,70 @@ id: Breadcrumb
...
+  <ListItem>
+    <Checkbox id="breadcrumb-a11y-checkbox-5" label={<span>The breacrumb has an <code className="ws-code">aria-label</code> if there are other <code className="ws-code">nav</code> elements or breacrumbs on the page.</span>} description="This helps differentiate the various navigations to users of assistive technologies." />
```
Comment (srambach) [inline packages/v4/patternfly-docs/content/accessibility/breadcrumb/breadcrumb.md:45]: Typo in "breadcrumb"
"Various navigations" sounds funny to me - maybe "This helps users of assistive technologies to differentiate between multiple navigational elements." ?

**Review by srambach — COMMENTED** [review]
Nice, just a couple of slight things 👍

**Review by srambach — APPROVED** [review]
Thanks! 👍

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**